tests/dummy/app/templates/index.hbs

```diff
@@ -261,3 +261,10 @@
     <strong>Warning!</strong> Better check yourself, you're not looking too good.
   {{/bsy-alert}}
 {{/bsy-sheet}}
+
+{{#bsy-sheet id="bsy-swal"}}
+  {{#bsy-sheet.header}}
+    <h2>Popups</h2>
```

Comment:
Let's change this title to <b>Alerts<b>

Reply:
There's already **Alerts** headline for `bsy-alerts` component above this section, can that be confusing?

Reply:
You're right. In my opinion those are not really alerts but more like info messages. Maybe we should rename those to something different?